tool.go

```diff
@@ -143,7 +143,11 @@ func main() {
 					}
 					if len(pkgs) == 1 { // Only consider writing output if single package specified.
 						if pkgObj == "" {
-							pkgObj = filepath.Base(pkg.ImportPath) + ".js"
+							if pkg.ImportPath == "." {
```

Comment:
I wonder if this would be a problem for the (much less common) case of `..` as well?

Why someone would `go build ..` I have no idea... :)  So whether to consider this is entirely up to you.

Reply:
Good catch. It's possible for this to happen, I verified. It'd be nice to fix it too.